src/main/java/com/neonlab/product/dtos/ProductRequestDto.java

```diff
@@ -0,0 +1,22 @@
+package com.neonlab.product.dtos;
+
+import com.neonlab.product.enums.Units;
+import lombok.Data;
+
+import java.math.BigDecimal;
+
+@Data
+public class ProductRequestDto {
```

Comment:
What is the difference between `productDto` and `productRequestDto`?

From implementation point of view it seems same.
cc- @sjha24

Reply:
tags extra points